pages/mobile/base.py

```diff
@@ -13,10 +14,11 @@ class Base(Page):
     _menu_items_locator = (By.CSS_SELECTOR, 'nav a')
     _menu_button_locator = (By.ID, 'menu-button')
     _mobile_header_locator = (By.CSS_SELECTOR, 'header.slide-on-exposed')
+    _search_bar_locator = (By.ID, 'search-bar')
 
     @property
     def is_menu_exposed(self):
-        return 'exposed' in self.selenium.find_element(*self._body_locator).get_attribute('class')
+        return self.selenium.find_element(*self._search_bar_locator).size['height'] == 270
```

Comment:
I suspect you need the open/close methods to wait for this to be true/false, but I don't see such waits.